i6_models/parts/conformer/mhsa.py

```diff
@@ -0,0 +1,41 @@
+import torch
+
+class ConformerMHSAV1(nn.Module):
```

Comment:
```suggestion
class ConformerMHSAV1(torch.nn.Module):
```
would this work without torch?